code/modules/admin/topic.dm

```diff
@@ -1993,14 +1993,19 @@
 		if(!istype(H))
 			to_chat(usr, "This can only be used on instances of type /mob/living/carbon/human")
 			return
-		if(!isLivingSSD(H))
+		if(!href_list["cryoafk"] && !isLivingSSD(H))
 			to_chat(usr, "This can only be used on living, SSD players.")
 			return
 		if(istype(H.loc, /obj/machinery/cryopod))
 			var/obj/machinery/cryopod/P = H.loc
 			P.despawn_occupant()
 			log_admin("[key_name(usr)] despawned [H.job] [H] in cryo.")
 			message_admins("[key_name_admin(usr)] despawned [H.job] [H] in cryo.")
+			if(href_list["cryoafk"]) // Warn them if they are send to storage and are AFK
+				to_chat(H, "<span class='danger'>The admins have moved you to cryo storage for being AFK. Please move out of cryostorage if you want to avoid being despawned.</span>")
+				H << 'sound/effects/adminhelp.ogg'

```

Comment:
shouldn't this use `SEND_SOUND`?

Reply:
Yep. Didn't realize it was there. Thanks for the tip!